CONTRIBUTING.md

```diff
@@ -21,3 +315,72 @@ Use [this guide][style] to maintain consistent style across the Conjur project.
 [style]: STYLE.md
 [tests]: README.md#Testing
 [issues]: https://github.com/cyberark/conjur/issues
+
+## Changelog maintenance
+
+The [changelog file](CHANGELOG.md) is maintained based on
+[Keep a Changelog](http://keepachangelog.com/en/1.0.0/) guidelines.
+
+Each accepted change to the Conjur code (documentation and website updates
+excepted) requires adding a changelog entry to the corresponding `Added`,
+`Changed`, `Deprecated`, `Removed`, `Fixed` and/or `Security` sub-section (add
+one as necessary) of the _Unreleased_ section in the changelog.
+
+Bumping the version number after each and every change is not required,
+advised nor expected. Valid reasons to bump the version are for example:
+
+- enough changes have accumulated,
+- an important feature has been implemented,
+- an external project depends on one of the recent changes.
+
+## Releasing
+
+### Verify and update dependencies
+1. Review the [NOTICES.txt](./NOTICES.txt) file and ensure it reflects the current
+   set of dependencies in the [Gemfile](./Gemfile)
+1. If a new dependency has been added, a dependency has been dropped, or a version
+   has changed since the last tag - make sure the NOTICES file is up-to-date with
+   the new versions
+
+### Update the version and changelog
+1. Examine the changelog and decide on the version bump rank (major, minor, patch).
+1. Change the title of _Unreleased_ section of the changelog to the target
+version.
+  - Be sure to add the date (ISO 8601 format) to the section header.
+1. Add a new, empty _Unreleased_ section to the changelog.
+  - Remember to update the references at the bottom of the document.
```

Comment:
Adding another two spaces to these bullets will set them under the numbered list item when rendered. They are currently showing up as indented with the numbered list items.